common/build/eslint-config-fluid/minimal-deprecated.js

```diff
@@ -8,7 +8,7 @@
  */
 const permittedImports = [
 	// Within Fluid Framework allow import of '/internal' from other FF packages.
-	"@fluidframework/*/internal",
+	"@fluid*/*/internal",
```

Comment:
Nit: given that our set of supported scopes is small, it might be safer to call each out here independently. Probably not super likely that we will encounter another `@fluid*` package out there, but just in case...

Reply:
Changed in latest!